test/controllers/v1/events_controller_test.rb

```diff
@@ -0,0 +1,64 @@
+require 'test_helper'
+
+class V1::EventsControllerTest < ActionController::TestCase
+  def setup
+    @logged_user = User.create(email: 'homer@simpsons.com', password: 'any-key', auth_token: 'stupid-flanders')
+    @other_user = User.create(email: 'bart@simpsons.com', password: 'el-barto', auth_token: 'skinner-stinks')
```

Comment:
Maybe is not required to have this User creation.
On the other hand `@other_xxx` seem not being used, so we don't need those instance vars.